doc/src/challenges/0003-pointer-arithmentic.md

```diff
@@ -0,0 +1,107 @@
+# Challenge 3: Verifying Raw Pointer Arithmetic Operations
+
+- **Status:** Open
+- **Solution:**
+- **Tracking Issue:** <https://github.com/model-checking/verify-rust-std/issues/21>
+- **Start date:** 24/06/24
+- **End date:** 24/12/10
```

Comment:
I should have said so when reviewing the template, perhaps you can fix it as part of this PR: can we please use ISO-8601 date format (YYYY-MM-DD)? There are a lot of 24s here, and one has to read both dates to get an idea which will be the months and the day.

Reply:
Do you want a separate PR for that?

Reply:
I'll create a separate PR